pkg/printers/internalversion/printers.go

```diff
@@ -41,6 +41,7 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/kubernetes/cmd/kubeadm/app/constants"
```

Comment:
This must not happen. Nothing in `pkg/` should depend on `cmd/kubeadm`
We must store the constant elsewhere, like https://github.com/kubernetes/kubernetes/pull/39112 tries to do
cc @jbeda @justinsb @philips